action.yml

```diff
@@ -34,12 +34,12 @@ runs:
         path: ${{ inputs.venv-dir }}
         key: setup-venv-${{ runner.os }}-py-${{ steps.setup-python.outputs.python-version }}-${{ steps.setup-python.outputs.python-path }}-${{ hashFiles(inputs.cache-dependency-path) }}-${{ inputs.install-cmd }}
 
-    - run: python3 -m venv ${{ inputs.venv-dir }}
+    - run: python -m venv ${{ inputs.venv-dir }}
       if: steps.cache-venv.outputs.cache-hit != 'true'
       shell: bash
 
     - run: |
-        source ${{ inputs.venv-dir }}/bin/activate
+        source ${{ inputs.venv-dir }}/${{ runner.os == 'Windows' && 'Scripts' || 'bin' }}/activate
         echo "VIRTUAL_ENV=${VIRTUAL_ENV}" >> $GITHUB_ENV
         echo "${VIRTUAL_ENV}/bin" >> $GITHUB_PATH
```

Comment:
🤔

Reply:
As per [PEP 405](https://peps.python.org/pep-0405/#creating-virtual-environments).

Reply:
I see you haven't read the line I commented on :)

Reply:
Oooh, right! Sorry. The highlighting took my focus away :)
This also needs a patch! Lemme fix it real quick.

Reply:
Probably fixed in [372cb53](https://github.com/getsentry/action-setup-venv/pull/12/commits/372cb5361613ac66733e506f1abf1e19936a16d0), I'll run tests locally.

Reply:
I [ran a test](https://github.com/bswck/action-setup-venv/actions/runs/10067852155/job/27832115928), seems to work, ready for another round.